### PR TITLE
Gh 3293/update safe settings

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #3290 #3293 #3294 #3295 

Changes proposed in this pull request:
- Create All screens related to MFA
- Tracking will be fixed in another pr 
- Navigation is not accurate, we need to integrate the SDK first

### CODE REVIEW

File: Multisig/UI/Settings/MFA/KeySecurityOverviewViewController.swift [hunk 1]
```diff
@@ -109,7 +110,7 @@ class KeySecurityOverviewViewController: LoadableViewController, UITableViewDele
         case .enabledFactors(_), .otherFactors(_):
             if case let Section.Factor.factor(name, value, image, isDefault, selected) = factor {
                 let cell = tableView.dequeueCell(SecurityFactorTableViewCell.self, for: indexPath)
-                cell.selectionStyle = .none
+                //cell.selectionStyle = .none
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/MFA/KeySecurityOverviewViewController.swift]: commented out

**Comment by DmitryBespalov** [comment]
pr is too big

File: Multisig/UI/Settings/SafeSettingsViewController/SafeSecurityViewController.swift [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard isValid(path: indexPath) else {
+            return
+        }
+        tableView.deselectRow(at: indexPath, animated: true)
+        let item = sections[indexPath.section].items[indexPath.row]
+        switch item {
+
+        case Section.RequiredConfirmations.confirmations(_):
+            changeConfirmationsFlow = ChangeConfirmationsFlow(safe: safe) { [unowned self] _ in
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/SafeSettingsViewController/SafeSecurityViewController.swift]: unowned reference

File: Multisig/UI/Settings/MFA/KeySecurityOverviewViewController.swift [hunk 3]
```diff
@@ -135,13 +135,20 @@ class KeySecurityOverviewViewController: LoadableViewController, UITableViewDele
...
-            present(flow: createPasswordFlow)
+        if indexPath.row == 0 {
+            show(TrustedEmailViewController(), sender: self)
+        } else if indexPath.row == 1 {
+            show(TrustedDeviceViewController(), sender: self)
+        } else {
+            let factor = sections[indexPath.section].items[indexPath.row]
+
+            if case let Section.Factor.factor(name, value, image, isDefault, selected) = factor {
+                createPasswordFlow = SetupRecoveryKitFlow(completion: { [weak self] _ in
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/MFA/KeySecurityOverviewViewController.swift:146]: the variable can be renamed, too

**Comment by DmitryBespalov** [comment]
If we leave it like this, you'll get the design bug issue , so here are some minor of the differences I found:

### Set up MFA



- [ ] missing "Learn More"

### Set Password


- [x] do we need the text under the title? (in figma there is a text)

 ### Success


- [x] title style is different from the design
- [x] the rows are different from the design - needs more space inside the borders

**Comment by MouazAlzahabi** [comment]
@DmitryBespalov all solved except the missing url, it's actually intended because we don't have the url yet, that's why the button is hidden

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**